src/main/java/ogc/rs/processes/collectionAppending/CollectionAppendingProcess.java

```diff
@@ -0,0 +1,412 @@
+                String outLog = stdout.toString();
+                String errLog = stderr.toString();
+                LOGGER.debug("ogr2ogr output: " + outLog);
+                LOGGER.debug("ogr2ogr error output: " + errLog);
+                future.complete();
+            } catch (IOException e) {
+                String errLog = stderr.toString();
+                LOGGER.error("ogr2ogr execution failed: " + errLog, e);
+                future.fail(e);
+            } catch (Exception exp) {
+                LOGGER.error("Exception Error: " + exp);
+                future.fail(exp); // Ensure the exception is propagated
+            }
+        }, VERTX_EXECUTE_BLOCKING_IN_ORDER).onSuccess(handler -> {
+            LOGGER.debug("Data appended successfully into temp table.");
+            promise.complete();
+        }).onFailure(promise::fail);
+
+        return promise.future();
+
+    }
+
+    private CommandLine getOgr2ogrCommandLine(JsonObject requestInput, String fileName) {
+
+        String jobId = requestInput.getString("jobId");
+        String tempTableName = "temp_table_for_" + jobId;
+
+        CommandLine cmdLine = new CommandLine("ogr2ogr");
+        cmdLine.addArgument("-nln");
+        cmdLine.addArgument(tempTableName);
+        cmdLine.addArgument("-lco");
+        cmdLine.addArgument("PRECISION=NO");
+        cmdLine.addArgument("-lco");
+        cmdLine.addArgument("LAUNDER=NO");
+        cmdLine.addArgument("-lco");
+        cmdLine.addArgument("GEOMETRY_NAME=geom");
+        cmdLine.addArgument("-lco");
+        cmdLine.addArgument("ENCODING=UTF-8");
+        cmdLine.addArgument("-lco");
+        cmdLine.addArgument("FID=id");
+
+        cmdLine.addArgument("-t_srs");
+        cmdLine.addArgument("EPSG:4326");
+        cmdLine.addArgument("--config");
+        cmdLine.addArgument("PG_USE_COPY");
+        cmdLine.addArgument("YES");
+        cmdLine.addArgument("-f");
+        cmdLine.addArgument("PostgreSQL");
+        cmdLine.addArgument(
+                String.format("PG:host=%s dbname=%s user=%s port=%d password=%s schemas=public", databaseHost,
+                        databaseName, databaseUser, databasePort, databasePassword), false);
+
+        cmdLine.addArgument("-progress");
+        cmdLine.addArgument("--debug");
+        cmdLine.addArgument("ON");
+
+        cmdLine.addArgument("--config");
+        cmdLine.addArgument("AWS_S3_ENDPOINT");
+        cmdLine.addArgument(awsEndPoint);
+        cmdLine.addArgument("--config");
+        cmdLine.addArgument("AWS_ACCESS_KEY_ID");
+        cmdLine.addArgument(accessKey);
+        cmdLine.addArgument("--config");
+        cmdLine.addArgument("AWS_SECRET_ACCESS_KEY");
+        cmdLine.addArgument(secretKey);
+
+        cmdLine.addArgument(String.format("/vsis3/%s%s", awsBucketUrl, fileName));
+        LOGGER.debug("cmdLine: " + cmdLine);
+        for (String arg : cmdLine.getArguments()) {
+            LOGGER.debug(arg);
+        }
```

Comment:
This can be omitted

Reply:
Omitted.
https://github.com/datakaveri/ogc-resource-server/pull/139/commits/4b99924bc8fd30100c9bb2a981498b159e23e6df